app/controllers/catalog_controller.rb

```diff
@@ -97,7 +97,7 @@ class CatalogController < ApplicationController
     config.add_facet_field 'geogname_sim', label: 'Place', limit: 10
     config.add_facet_field 'places_ssim', label: 'Places', show: false
     config.add_facet_field 'access_subjects_ssim', label: 'Subject', limit: 10
-
+    config.add_facet_field 'campus_unit_sim', label: 'Campus', helper_method: :render_campus_facet
```

Comment:
```suggestion
    config.add_facet_field 'campus_unit_sim', label: 'Campus', helper_method: :render_campus_facet
    
```
can we add the linebreak back to make the separation clearer?

Reply:
I don't understand this suggestion

Reply:
You replaced the one line of whitespace between that set of code and the next one

Reply:
I.e. it went from
```
  config.add_facet_field 'access_subjects_ssim', label: 'Subject', limit: 10

 # Have BL send all facet field names to Solr, which has been the default
```

to 

```
    config.add_facet_field 'access_subjects_ssim', label: 'Subject', limit: 10
    config.add_facet_field 'campus_unit_sim', label: 'Campus', helper_method: :render_campus_facet
    # Have BL send all facet field names to Solr, which has been the default
```